### PR TITLE
feat: hamburguer menu for sm and md screens

### PR DESCRIPTION
**Pull request by Cauatn**
Think it fits nice

image

image

### CODE REVIEW

File: src/components/global/hamburguer.astro [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<div class="text-white">
+  <button id="open" class="w-auto h-auto lg:hidden">
+    <img
+      src="../../public/assets/img/hamburguer-menu.svg"
+      alt="Hamburguer Menu"
```
Comment (AtkinsSJ) [inline src/components/global/hamburguer.astro:5]: Typo in alt text, this file's name, and the image's filename: "hamburger"

**Comment by Kingproone** [comment]
https://github.com/LadybirdBrowser/ladybird.org/issues/6 just so it auto closes on merge.

File: src/components/global/hamburguer.astro (hunk 1 above)
Comment (Cauatn) [reply src/components/global/hamburguer.astro]: fixed !

**Comment by Cauatn** [comment]
closing to open a new clean pr.

**Pull request closed without merging**